submissions/Nutan/index.html

```diff
@@ -0,0 +1,23 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>document</title>
+</head> 
+<body style="margin-left: 20px;margin-top:20px;margin-right: 20px; background-color: paleturquoise">
```

Comment:
You are not expected to use any CSS styling for this task. Kindly go through and make changes in other places too.